omim2obo/main.py

```diff
@@ -231,18 +243,53 @@ def run(use_cache: bool = False):
-            graph.add((OMIM[mim_number], RO['0003303'], OMIM[phenotype_mim_number]))
+    # todo: when addressed, remove this from the issue report JSON (or remove the report entirely)
+    morbid_map: Dict = parse_morbid_map(get_mim_file('morbidmap.txt', download_files_tf))
+    for mim_number, mim_data in morbid_map.items():
+        # todo?: unused `mim_data` keys:
+        #   'gene_mim_number': Should this be used somewhere?
+        #   'gene_symbols':  Should this be used somewhere?
+        #   'phenotype_label': I think this already added to graph because the MIM# and label shared in other file(s)
+        cyto_location: str = mim_data['cyto_location']
         if cyto_location:
             # What's 9606chr - joeflack4 2021/11/11
             chr_id = '9606chr' + cyto_location
-            # What's RO['0002525'] - joeflack4 2021/11/11
+            # RO:0002525 (is subsequence of)
+            # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
             graph.add((OMIM[mim_number], RO['0002525'], CHR[chr_id]))
+        assocs: List[Dict] = mim_data['phenotype_associations']
+        for assoc in assocs:
+            # RO:0003303 (causes condition)
+            # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0003303
+            p_mim, p_lab, p_map_key, p_map_lab = \
+                assoc['phenotype_mim_number'], assoc['phenotype_label'], assoc['phenotype_mapping_info_key'], \
+                assoc['phenotype_mapping_info_label']
+            # i. Add to MIM
+            b = BNode()
+            graph.add((b, RDF['type'], OWL['Restriction']))
+            graph.add((b, OWL['onProperty'], RO['0003303']))
+            if p_mim:
+                graph.add((b, OWL['someValuesFrom'], OMIM[p_mim]))
+            else:
+                graph.add((b, OWL['someValuesFrom'], Literal(p_lab)))
+            graph.add((OMIM[mim_number], RDFS['subClassOf'], b))
+
+            # ii. add axiom
+            b2 = BNode()
+            graph.add((b2, RDF['type'], OWL['Axiom']))
+            graph.add((b2, OWL['annotatedSource'], OMIM[mim_number]))
+            graph.add((b2, OWL['annotatedProperty'], RDFS['subClassOf']))
+            # todo: add biolink evidence/genediseaseassoc?
+            # todo: add mapping key and related info in a different way?
+            graph.add((b2, RDFS['comment'], Literal(f'Evidence: ({p_map_key}) {p_map_lab}')))
+            b3 = BNode()
+            graph.add((b3, RDF['type'], OWL['Restriction']))
+            graph.add((b3, OWL['onProperty'], RO['0003303']))
+            if p_mim:
+                graph.add((b3, OWL['someValuesFrom'], OMIM[p_mim]))
+            else:
+                graph.add((b3, OWL['someValuesFrom'], Literal(p_lab)))
+            graph.add((b2, OWL['annotatedTarget'], b3))
```

Comment:
Stilling missing the evidence annotations on top?

Reply:
@matentzn Actually, I'm not sure where you want me to add this. To the axiom, along with the comment about the evidence, or directly to the class. I will add it to the axiom since that is more consistent. So it will look like:

```ttl
OMIM:100678 a owl:Class ;
    rdfs:label "ACAT2" ;
    ...
    rdfs:subClassOf [ a owl:Restriction ;
            owl:onProperty RO:0003303 ;
            owl:someValuesFrom OMIM:614055 ],
        SO:0000704 ;
    biolink:category biolink:Gene .

[] a owl:Axiom ;
    biolink:has_evidence "Evidence: (1) The disorder is placed on the map based on its association with a gene, but the underlying defect is not known."   # <------------------------ this is the new addition
    rdfs:comment "Evidence: (1) The disorder is placed on the map based on its association with a gene, but the underlying defect is not known." ;
    owl:annotatedProperty rdfs:subClassOf ;
    owl:annotatedSource OMIM:100678 ;
    owl:annotatedTarget [ a owl:Restriction ;
            owl:onProperty RO:0003303 ;
            owl:someValuesFrom OMIM:614055 ] .
```

Reply:
@matentzn Actually, I'm not sure where you want me to add this. To the axiom, along with the comment about the evidence, or directly to the class. I will add it to the axiom since that is more consistent. So it will look like:

```ttl
OMIM:100678 a owl:Class ;
    rdfs:label "ACAT2" ;
    ...
    rdfs:subClassOf [ a owl:Restriction ;
            owl:onProperty RO:0003303 ;
            owl:someValuesFrom OMIM:614055 ],
        SO:0000704 ;
    biolink:category biolink:Gene .

[] a owl:Axiom ;
    biolink:has_evidence "Evidence: (1) The disorder is placed on the map based on its association with a gene, but the underlying defect is not known."   # <------------------------ this is the new addition
    rdfs:comment "Evidence: (1) The disorder is placed on the map based on its association with a gene, but the underlying defect is not known." ;
    owl:annotatedProperty rdfs:subClassOf ;
    owl:annotatedSource OMIM:100678 ;
    owl:annotatedTarget [ a owl:Restriction ;
            owl:onProperty RO:0003303 ;
            owl:someValuesFrom OMIM:614055 ] .
```

Reply:
Yes definitely to the axiom!